apps/azure-devops/azure-devops-agent-keda/azure-devops-agent.yaml

```diff
@@ -9,6 +9,9 @@ spec:
     scaleType: Job
     image: hmctspublic.azurecr.io/azure-devops-agent:prod-cb09685d-1678281568 # {"$imagepolicy": "flux-system:azure-devops-agent"}
     runAsRoot: true
+    aadIdentity:
```

Comment:
Will this work? Sorry just checking the charts and the original azure devops agent uses https://github.com/hmcts/sds-helm-charts/blob/master/charts/azure-devops-agent/azure-devops-agent/templates/deployment.yaml#L32 with the addpodidentity binding set.
The keda one uses https://github.com/hmcts/chart-function/tree/master I believe, which does seem to have some sort of triggerPodIdentityProvider: Azure setting, but nowhere to actually define the managed identity client id being used?